src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 0;
+
+  if (days < 7) {
+    totalAmount = days * 40;
```

Comment:
move main data  to variables

Reply:
sorry I do not understand

Reply:
not bacause of English just dont understand what to move where